scripts/docker/update_json_docker.sh

```diff
@@ -68,7 +68,7 @@ JSONS=$(printf "%s "  "${JSON_ARR[@]}")
 for name in "${DOCKER_NAME_ARR[@]}"; do
   docker_var=$(tr '-' '_' <<< "${name}")"_docker"
   docker_regex='\.'"${docker_var}"'"\s*:\s*".+\/.+:.+"'
-  cmd="perl -pi -e 's/${docker_regex}/.${docker_var}\": \"${DOCKER_ROOT}\/${name}:${DOCKER_TAG}\"/g' ${JSONS}"
+  cmd="perl -pi -e 's|${docker_regex}|.${docker_var}\": \"${DOCKER_ROOT}\/${name}:${DOCKER_TAG}\"|g' ${JSONS}"
```

Comment:
Just curious what this does?

Reply:
I just wanted to allow using `DOCKER_ROOT`s with forward slashes in their names, which we need to do to this script with gcr repo identifiers. Changing the `s///` command delimiter to `s|||` lets you put in forward slashes without escaping them, so you can invoke this script like this:

`scripts/docker/update_json_docker.sh -d . -r us.gcr.io/broad-dsde-methods/cwhelan -n sv-pipeline -t cw_ctx_fixes_d1e9818`